definitions/features/dynflow_sidekiq.rb

```diff
@@ -13,7 +13,7 @@ def services
 
   def config_files
     # Workaround until foreman-installer can deploy scaled workers
-    service_symlinks = configured_services.map do |service|
+    service_symlinks = services.map do |service|
```

Comment:
I'm pretty sure it should use `configured_instances` instead.

Reply:
Fixed with `configured_instances`!